src/client/Microsoft.Identity.Client/Platforms/Android/Broker/AndroidBrokerHelper.cs

```diff
@@ -29,26 +36,34 @@ internal class AndroidBrokerHelper
     {
         private const string RedirectUriScheme = "msauth";
         private const string BrokerTag = BrokerConstants.Signature;
-        public const string WorkAccount = "com.microsoft.workaccount.user.info";
 
         private readonly Context _androidContext;
 
         // Important: this object MUST be accessed on a background thread. Android will check this and throw otherwise.
-        private readonly AccountManager _androidAccountManager;
+        public AccountManager AndroidAccountManager { get; }
         private readonly ICoreLogger _logger;
 
-        private const long AccountManagerTimeoutSeconds = 5 * 60;
-
         public AndroidBrokerHelper(Context androidContext, ICoreLogger logger)
         {
             _androidContext = androidContext ?? throw new ArgumentNullException(nameof(androidContext));
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
 
             _logger.Verbose("Getting the Android context for broker request. ");
-            _androidAccountManager = AccountManager.Get(_androidContext);
+            AndroidAccountManager = AccountManager.Get(_androidContext);
```

Comment:
>AccountManager [](start = 36, length = 14)

I get a warning that this `AccountManager` is only available on Android 9, not 10.

Reply:
Ill double check the availability

Reply:
I checked this out and can confirm that this is accessible when targeting android 10. Might be a VS problem on your end.